File: components_page/components/alert/link.py
import dash_bootstrap_components as dbc
from dash import html

alerts = html.Div(
    [
        dbc.Alert(
            [
                "This is a primary alert with an ",
                html.A("example link", href="#", className="alert-link"),
            ],
            color="primary",
        ),
        dbc.Alert(
            [
                "This is a danger alert with an ",
                html.A("example link", href="#", className="alert-link"),
            ],
            color="danger",
        ),
    ]
)

File: components_page/components/alert/link.jl
using DashBootstrapComponents, DashHtmlComponents

alerts = html_div([
    dbc_alert(
        [
            "This is a primary alert with an ",
            html_a("example link", href = "#", className = "alert-link"),
        ],
        color = "primary",
    ),
    dbc_alert(
        [
            "This is a danger alert with an ",
            html_a("example link", href = "#", className = "alert-link"),
        ],
        color = "danger",
    ),
]);
